src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInMdx.ts

```diff
@@ -32,8 +34,8 @@ export const migrateCalculatedMeasuresInMdx = (
       removeCalculatedMemberDefinition(acc, {
         dimensionName: "Measures",
         hierarchyName: "Measures",
-        namePath: [calculatedMeasureName],
-        cube: getCube(dataModel, cubeName),
+        calculatedMemberName: calculatedMeasureName,
+        cube: getCube(getIndexedDataModel(dataModel), cubeName),
```

Comment:
Can you add a TODO to pass the indexed `dataModel` to the function directly ?
We'll wire it when we're finished with the refactoring how to generically migrate from 5.x to 5.y